### PR TITLE
Scanpy and Anndata minor fixes to labels

### PR DESCRIPTION
**Pull request by mtekman**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Mirrors fixes to the PBMC 3K tutorial here: https://github.com/galaxyproject/training-material/pull/1751

### CODE REVIEW

**Comment by bgruening** [comment]
Can you please also fix the linting of the failing URL?

File: tools/scanpy/cluster_reduce_dimension.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<tool id="scanpy_cluster_reduce_dimension" name="Cluster," version="@galaxy_version@" profile="@profile@">
-    <description>infer trajectories and embed with scanpy</description>
+<tool id="scanpy_cluster_reduce_dimension" name="Cluster, infer trajectories and embed" version="@galaxy_version@" profile="@profile@">
```
Comment (wm75) [inline tools/scanpy/cluster_reduce_dimension.xml]: I guess a change of the tool's name warrants a wrapper version bump?

Comment (bgruening) [reply tools/scanpy/cluster_reduce_dimension.xml]: I don't think so. The user will either way only see one tool and get the new name in both cases.

**Comment by mtekman** [comment]
thanks guys, I didn't bump because it didn't look like a feature upgrade

File: tools/scanpy/cluster_reduce_dimension.xml (hunk 1 above)
Comment (wm75) [reply tools/scanpy/cluster_reduce_dimension.xml]: That's true for any given server. On *different* servers, however, you can now have the same tool version under two different names, which may leave users wondering whether these are equivalent.

Anyway, this is not particularly critical though :)

**Pull request merged**